Library/Homebrew/utils/formatter.rb

```diff
@@ -51,7 +51,7 @@ def prefix(prefix, string, color)
   private_class_method :prefix
 
   def columns(*objects, gap_size: 2)
-    objects = objects.flatten.map(&:to_s)
+    objects = objects.flat_map(&:to_s)
```

Comment:
`arr.flat_map` is equivalent to `arr.map.flatten`, *not* `arr.flatten.map`, unfortunately.